include/podio/BenchmarkRecorder.h

```diff
@@ -35,32 +34,27 @@ class BenchmarkRecorderTree {
     }
   }
 
-  template<typename TimingResolution=std::chrono::nanoseconds>
+  template <typename TimingResolution = std::chrono::nanoseconds>
   void recordTime(const std::string& stepName, const ClockT::duration time) {
     const auto it = std::find(m_stepNames.cbegin(), m_stepNames.cend(), stepName);
     const auto index = std::distance(m_stepNames.cbegin(), it);
     m_stepTimes[index] = std::chrono::duration_cast<TimingResolution>(time).count();
   }
 
-  void Fill() {
-    m_recordTree->Fill();
-  }
+  void Fill() { m_recordTree->Fill(); }
```

Comment:
I agree and this controlled by the `AllowShortFunctionsOnASinlgeLine` option ([all options](https://clang.llvm.org/docs/ClangFormatStyleOptions.html), unfortunately there seems to be no way to link directly to one, so a Ctrl+F is still necessary).

There is the possibility to have this enabled if it appears inside class definitions (`InlineOnly` or `Inline`) but have it disabled outside, which would make it possible to have "trivial" single line calls on the same line class definitions. If more than that is happening in such a call one could make an argument to move it to a dedicated implementation in any case. It can also be disabled everywhere (`None`). I have no strong preference here on whether we want to allow this inside class definitions.